app/lnd/subscribe/channelgraph.js

```diff
@@ -1,8 +1,14 @@
-export default function subscribeToChannelGraph(mainWindow, lnd) {
+import { status } from 'grpc'
+
+export default function subscribeToChannelGraph(mainWindow, lnd, log) {
   const call = lnd.subscribeChannelGraph({})
 
   call.on('data', channelGraphData => mainWindow.send('channelGraphData', { channelGraphData }))
+  call.on('end', () => log.info('end'))
+  call.on('error', error => (error.code === status.CANCELLED ? null : log.error(error)))
```

Comment:
this can be simplified a bit because you're not utilizing the front half of the conditional:

`call.on('error', error => (error.code !== status.CANCELLED && log.error(error)))`